ci/Jenkinsfile.combined

```diff
@@ -1,4 +1,4 @@
-library 'status-jenkins-lib@v1.3.3'
+library 'status-jenkins-lib@6328a0484c8cd331ed8ef58e8ae1400cfaff08aa'
```

Comment:
You can just use the branch name, like:
```
library 'status-jenkins-lib@creds/opensea-api-key'
```
But this works too. Once we confirm this works correctly I'll merge that PR and we can use a tag here.